### PR TITLE
New Decorator testing

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
Test for new decorator

### CODE REVIEW

File: VulnerableApplication/Global.asax.cs [hunk 1]
```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public const string Password = "abc";
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWYVm1kdWX7Ifb7k&pullRequest=15&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWYVm1kdWX7Ifb7l&pullRequest=15&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWYVm1kdWX7Ifb7m&pullRequest=15&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **VULNERABILITY**
> 'password' detected in this expression, review this potentially hardcoded credential.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWYVm1kdWX7Ifb7n&pullRequest=15&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Change this constant to a 'static' read-only property.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWYVm1kdWX7Ifb7o&pullRequest=15&resolved=false)

**Comment by TsuyoshiUshio** [comment]
**CODE_SMELL**
> [VulnerableApplication] Mark this assembly with 'System.CLSCompliantAttribute'
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWurWWZUm1kdWX7Ifb7p&pullRequest=15&resolved=false)

File: VulnerableApplication/Global.asax.cs (hunk 1 above)
Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

**Comment by TsuyoshiUshio** [comment]
kokokara
